.github/workflows/tests.yml

```diff
@@ -74,24 +74,27 @@ jobs:
           composer update --no-interaction --prefer-dist
 
       - name: Setup Mink test server
+        env:
+          MINK_HOST: 0.0.0.0:8002
```

Comment:
is this still needed then?

Reply:
Yes, because otherwise Web Server would start on `localhost` but be accessed from within Docker container on `172.17.0.1`.

The similar approach is taken (replacement of `localhost` with `0.0.0.0`) in https://github.com/minkphp/webdriver-classic-driver/blob/main/tests/bootstrap.php#L13 .

Reply:
AAah, I see.. this is listening host, not the connect-to host. All good then.